persist/fs/files_test.go

```diff
@@ -165,27 +165,29 @@ func TestDeleteFiles(t *testing.T) {
 }
 
 func TestDeleteInactiveFiles(t *testing.T) {
-	tempPrefix := "temp/"
+	tempPrefix, err := ioutil.TempDir("", "foo")
+	require.NoError(t, err)
+	defer func() {
```

Comment:
so is this the trick to getting it to eval later?

Reply:
`temp` would've only been available if you made it on your own machine, `ioutil.TempDir(...)` creates a temporary directory for use in a directory the stdlib manages.